From 4f5992a6d6fec88bcef665f7a87887872c05cde5 Wed Feb 25 14:33:13 2026
From: Valeriu Predoi <valeriu.predoi@gmail.com>
Date: Wed, Feb 25 14:33:13 2026 +0000
Subject: [PATCH] add core dependency

---
 pyproject.toml | 1 +
 environment.yml | 1 +
 2 files changed, 2 insertions(+)

diff --git a/environment.yml b/environment.yml
index 3e95a5f..c48bdd6 100644
--- a/environment.yml
+++ b/environment.yml
@@ -8,6 +8,7 @@ dependencies:
   # core dependencies
   - numpy >=2
   - python >=3.10
+  - typing_extensions
   # testing dependencies
   - dask
   - flask
diff --git a/pyproject.toml b/pyproject.toml
index d12a550..64d5c8f 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -37,6 +37,7 @@ dynamic = [
 ]
 dependencies = [
     "numpy>=2",
+    "typing-extensions",
 ]
 description = "A pure python HDF5 reader"
 license = "BSD-3-Clause"
--
2.51.0
